docs/architecture/tokens.md

```diff
@@ -5,32 +5,35 @@ last_update:
   date: 28 November 2023
 ---
 
-In a blockchain ecosystem, a digital asset that can be transferred between accounts is called a _token_.
-Tezos supports many types of tokens, including:
+Any blockchain ecosystem relies on a native token that can be transferred between accounts and in which transaction fees are paid in.
+The native token of Tezos is tez (also known as XTZ or represented by the symbol ꜩ).
+
+But other tokens representing some value in digital for can be programmed in a blockchain, for instance using smart contracts.
+Tokens fall in two broad categories:
+
+- Fungible tokens, whose instances are interchangeable and represent the same value,
```

Comment:
```suggestion
- Fungible tokens, which are interchangeable and represent the same value
```
"Whose instances" is wordy; how about just "which?"

Reply:
thanks, applied